src/modules/profile/public/EditPublicProfile.tsx

```diff
@@ -120,6 +120,8 @@ const EditPublicProfile = ({
   };
 
   useEffect(() => {
+    // TODO: remove need for synching state
```

Comment:
```suggestion
    // TODO: need to remove to sync state
```
Is is still a todo? Not sure to understand this comment tho.

Reply:
I have tracked this in #401. We need to refactor the code to suppress the usage of states and useEffects.